icekit/bin/entrypoint.sh

```diff
@@ -31,15 +31,23 @@ if [[ -n "$DOCKER" ]]; then
     export -f pip
 
     # Use alternate installation (user scheme) for Python packages.
-    export PIP_SRC="$ICEKIT_PROJECT_DIR/var/venv/src"
-    export PYTHONUSERBASE="$ICEKIT_PROJECT_DIR/var/venv"
+    export PYTHONUSERBASE="$ICEKIT_PROJECT_DIR/var/docker-pyuserbase"
+    export PIP_SRC="$PYTHONUSERBASE/src"
+
+    # Add user scheme bin directory to PATH.
+    export PATH="$PYTHONUSERBASE/bin:$PATH"
 
     # For some reason pip allows us to install sdist packages, but not editable
     # packages, when this directory doesn't exist. So make sure it does exist.
     mkdir -p "$PYTHONUSERBASE/lib/python2.7/site-packages"
 else
     # Add ICEkit bin directory to PATH.
     export PATH="$ICEKIT_DIR/bin:$PATH"
+
+    # Prefer ICEKit virtualenv binaries to run things inside the venv
+    if [[ -n "$ICEKIT_VENV" ]]; then
+        export PATH="$ICEKIT_VENV/bin:$PATH"
+    fi
```

Comment:
Fair points. I now agree it's reasonable to require `pv` and failing if it isn't present with a useful message should be enough to avoid any confusion